dietpi/dietpi-software

```diff
@@ -14864,6 +14907,30 @@ _EOF_
 			[[ -d '/mnt/dietpi_userdata/docker-data' ]] && rm -R /mnt/dietpi_userdata/docker-data
 			# Remove default unused folder
 			[[ -d '/var/lib/docker' ]] && rm -R /var/lib/docker
+			# Remove configuration file   
+			[[ -f '/etc/docker/daemon.json' ]] && rm -R /etc/docker/
```

Comment:
I guess the empty dir (or if anything else is contained) could be removed as well:
```
[[ -d '/etc/docker' ]] && rm -R /etc/docker
```

Reply:
ok going to check for the directory and remove it if existing